### ISSUE TITLE
plugin.xml should handle extension points from plugins other than org.eclipse.ui

### ISSUE THREAD

**Issue by deepamin-intel** [JaneWardSandy/eclipse-pde-partial-idea#53]
**Is your feature request related to a problem? Please describe.**
Extension points from CDT plugins are reported as error as seen in attached screenshot, which shouldn't be the case

**Describe the solution you'd like**
Extension points and associated attributes with those points should not be reported as error and would be good to have autocomplete on them as with org.eclipse.ui extension points.

![image](https://user-images.githubusercontent.com/77679096/129056416-2f2830a9-5e57-4102-b039-31e59c60e3c6.png)

**Comment by JaneWardSandy** [comment]
We make autocomplete from extension point exsd file, and this file we need to read plugin.xml first, and get the `extension-point` tag, get the EP name, and schema location, then read exsd file, then make autocomplete for plugin.xml.

If we can not found `extension-point` from plugin.xml in bundles we checked in `eclipse target`, or schema file not found in bundle or source bundle, it will mark as invalid as you see.

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?eclipse version="3.0"?>
<plugin>
     <extension-point id="CDebuggerPage" name="%CDebuggerPage" schema="schema/CDebuggerPage.exsd"/>
     <extension-point id="BreakpointActionPage" name="%BreakpointActionPage" schema="schema/BreakpointActionPage.exsd"/>
     <extension-point id="breakpointContribution" name="%breakpointContribution" schema="schema/BreakpointUIContribution.exsd"/>
     <extension-point id="disassemblyRulerColumns" name="%extPoint.disassemblyRulerColumn" schema="schema/disassemblyRulerColumns.exsd"/>
</plugin>
```

**Comment by deepamin-intel** [comment]
yes, that was the reason since I had added the dependencies manually I hadn't added the source packages along side.
